docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,125 @@
+
+To improve the readability of the CSV file, some code editors, such as Visual Code, have extensions to provide a better visualization from the CSV file:
+
+- [Edit CSV](https://marketplace.visualstudio.com/items?itemName=janisdd.vscode-edit-csv)
+- [CSV rainbow](https://marketplace.visualstudio.com/items?itemName=mechatroner.rainbow-csv)
+- [CSV to table](https://marketplace.visualstudio.com/items?itemName=phplasma.csv-to-table)
+
+You can fork the GitHub project to start editing and creating your branches. Once you've filled in the information about the cookie, you'll be required to make a pull request to be evaluated by the repository owner. In the following section, we will explain step-by-step how you can set up the project.
+
+## Setup repository on your local
+Once you have your GitHub account, you must fork the Open Cookie Database repository. This step will create a repository with the same code and visibility in your account.
+
+To fork, click on the fork button in the top-right corner of the upstream repository.
+
+<img src="https://i.ibb.co/XsdGHQ9/github-repo-open-cookie-db.png" alt="Open cookie database repository" width="640">
+
+Once your fork is ready, you must clone that repository to your computer. You can do this using the git command, or if you are not tech-savvy, you can use the GitHub application, which provides a GUI for ease of use.
+
+We will discuss how you can  use the command line in the following process:
+
+Open the terminal on Mac or PowerShell on Windows.
+1. Enter the below command to clone the repository:
+2. `$ git clone git@github.com:yourgithubname/Open-Cookie-Database.git`
+
+<img src="https://i.ibb.co/z8WGHCX/github-clone-repo-open-Cookie.png" alt="Clone repository on Github" width="640">
+
+1. Create a new branch from the master branch `$ git checkout -b cookie-vendor-name`
+2. And now, you are ready to collaborate with the Open Cookie database
+3. Open the file open-cookie-database.csv and start to contribute.
+
+## Editing
+Once you are ready, remember to create a branch, for example, `cookie-[vendor-name]` to submit your changes and start editing the file. The first information is a UUID to register the new cookie. A unique ID creates this information. You can find it online at websites like [uuid generator](https://www.uuidgenerator.net/) or use the node package UUID: `$ npx uuid`. 
+
+Add the related information to the cookie, and two fields could have special attention. The cookie name could have a dynamic ID, for example, `cookie_[site-id]`. If this applies to your case, the last property on the cookie is a wildcard. For wildcard cookies, set the value to 1 instead; the dynamic property doesn’t need to be specified. For example, we set the previous cookie as `cookie_`.
+
+<img src="https://i.ibb.co/Nn63CmP/open-cookie-db-vscode.png" alt="Open Cookie database on VS Code" width="640">
+
+In another scenario, some solutions set the cookies as first-party; for this scenario, it can be empty.
```

Comment:
Can we please also add? `How to organize the cookies` sub section

Reply:
Did you pull the latest version? Now, is called Organizing Cookies